metricbeat/docs/fields.asciidoc

```diff
@@ -29819,6 +29819,23 @@ alias to: elasticsearch.node.stats.thread_pool.bulk.rejected.count
 
 --
 
+*`node_stats.thread_pool.force_merge.queue`*::
```

Comment:
I don't see these aliases defined in the mappings which is probably what is causing the build to fail.
For context, these aliases exist to make Stack Monitoring application querying easier and if we have no plan to use them there we can remove them.

Reply:
🪄  it is gone ^^

Reply:
But I believe this is not what make the build fail.
From the build logs it is `Error: some files are not up-to-date. Run 'make update'` which I am aware of but for review purposes I felt it was better not to include those changes **yet**.